### DISCUSSION TITLE
urltransform

### DISCUSSION THREAD

**Discussion by MasterKia** [uBlockOrigin/uBlock-issues#2923] [Ideas]
I want to change `https://upera.shop/ref/3m8/ckz0oa1m` to `https://upera.shop/ckz0oa1m` to prevent tracking, how can I do it using `$urltransform`?

**Comment by stephenhawk8054** [comment]
Is it `doc`?

**Reply by MasterKia** [reply]
Yes, they're the download links here:
`https://www.doostihaa.com/post/1402/08/09/%d9%82%d8%b3%d9%85%d8%aa-18-%d9%85%da%af%d9%87-%d8%aa%d9%85%d9%88%d9%85-%d8%b9%d9%85%d8%b1-%da%86%d9%86%d8%af-%d8%aa%d8%a7-%d8%a8%d9%87%d8%a7%d8%b1%d9%87.html`

**Reply by stephenhawk8054** [reply]
Maybe this?

```adb
||upera.shop/ref/3m8/ckz0oa1m$doc,urltransform=/ckz0oa1m
```

But I think you want regex and `$1` as well?

**Reply by MasterKia** [reply]
Yes, the `3m8` and `ckz0oa1m` parts will differ.

`3m8` is the tracking code for each site.
`ckz0oa1m` is the video code.

**Reply by stephenhawk8054** [reply]
Hmm... I'm afraid regex is not supported yet

**Reply by MasterKia** [reply]
https://github.com/uBlockOrigin/uBlock-issues/issues/2924
